src/presubmit-service/check-pr.ts

```diff
@@ -5,7 +5,10 @@ export const getCherryPickConflictStatus =
     (org: string, repo: string, pr: string, branch: string) => {
   // TODO(josephperrott): Use stderr/stdout output from the script to fill in the message property
   // values in the returned object.
-  const prCheck = shell.exec(`./check_if_pr_can_cherry_pick.sh ${org} ${repo} ${pr} ${branch}`);
+  // TODO(josephperrott): Determine how to not have to know where to expect the script to magically
+  // exist.
+  const prCheck = shell.exec(
+    `./dist/src/presubmit-service/check_if_pr_can_cherry_pick.sh ${org} ${repo} ${pr} ${branch}`);
```

Comment:
You could use `__dirname` right?

Reply:
Yeah, after you started your review I realized that.